src/pages/index.js

```diff
@@ -31,56 +63,114 @@ const popupEditProfileForm = new PopupWithForm(
   ".modal_profile-edit",
   handleProfileEditSubmit
 );
-const gallerySection = new Section(
-  { items: initialCards, renderer: cardRenderer },
-  cardGalleryEL
+const popupAvatarForm = new PopupWithForm(
+  ".modal_change-avatar",
+  handleAvatarFormSubmit
 );
-
-//////////////////////////  functions  //////////////////////////
+////////////////////////////////////////////////////////////
+///////////////////////   API    ///////////////////////////
+////////////////////////////////////////////////////////////
+const apiCall = new Api(
+  "https://around-api.en.tripleten-services.com/v1",
+  serverToken
+);
+////////////////////////////////////////////////////////////
+///////////////////////   FUNCTIONS    /////////////////////
+////////////////////////////////////////////////////////////
+function handleDeleteFormSubmit(data) {
+  apiCall
+    .deleteCard(data.data)
+    .then(data.cardElement.remove())
```

Comment:
`    .then(data.cardElement.remove())
`
In blocks `then`, `catch`, `finally` you need to pass only the **reference** to the function (not a function call) otherwise the function is called immediately without waiting for the response from the server. You can pass an **unnamed** arrow function as well.